content/id/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,374 @@
+untuk sekumpulan tingkat prioritas (_priority levels_) yang dapat dikonfigurasi. Setiap permintaan masuk diserahkan
+ke sebuah tingkat prioritas, dan setiap tingkat prioritas hanya akan meneruskan sebanyak mungkin
+permintaan secara bersamaan sesuai dengan yang diijinkan dalam konfigurasi.
+
+Konfigurasi bawaan, misalnya, sudah mencakup tingkat prioritas terpisah untuk
+permintaan dalam rangka pemilihan pemimpin (_leader-election_), permintaan dari _controller_ bawaan, dan permintaan dari
+Pod. Hal ini berarti bahwa Pod yang berperilaku buruk, yang bisa membanjiri server API
+dengan permintaan, tidak akan mampu mencegah kesuksesan pemilihan pemimpin atau tindakan yang dilakukan oleh _controller_ bawaan.
+
+### Antrian (_Queuing_)
+
+Bahkan dalam sebuah tingkat prioritas mungkin akan ada sumber lalu lintas yang berbeda dalam jumlah besar. 
+Dalam situasi beban berlebih, sangat penting untuk mencegah satu aliran
+permintaan dari penderitaan karena aliran yang lainnya (khususnya, dalam kasus yang relatif umum dari sebuah
+klien tunggal bermasalah (_buggy_) yang dapat membanjiri _kube-apiserver_ dengan permintaan, klien bermasalah itu
+idealnya tidak memiliki banyak dampak yang bisa diukur terhadap klien yang lainnya). Hal ini 
+ditangani dengan menggunakan algoritma _fair-queuing_ untuk memproses permintaan yang diserahkan
+oleh tingkat prioritas yang sama. Setiap permintaan diserahkan ke sebuah _flow_, yang diidentifikasi berdasarkan
+nama _FlowSchema_ yang sesuai, ditambah dengan _flow distinguisher_ - yang
+bisa saja didasarkan pada pengguna yang meminta, sumber daya _namespace_ dari target, atau tidak sama sekali - dan
+sistem mencoba untuk memberikan bobot yang hampir sama untuk permintaan dalam _flow_ yang berbeda dengan tingkat prioritas yang sama.
+
+Setelah mengklasifikasikan permintaan ke dalam sebuah _flow_, fitur APF kemudian 
+dapat menyerahkan permintaan ke dalam sebuah antrian. Penyerahan ini menggunakan
+teknik yang dikenal sebagai {{<glossary_tooltip term_id="shuffle-sharding"
+text="_shuffle sharding_">}}, yang membuat penggunaan antrian yang relatif efisien
+untuk mengisolasi _flow_ dengan intensitas rendah dari _flow_ dengan intensitas tinggi.
+
+Detail dari algoritma antrian dapat disesuaikan untuk setiap tingkat prioritas, dan
+memperbolehkan administrator untuk menukar (_trade off_) dengan penggunaan memori, kesetaraan (properti dimana
+_flow_ yang independen akan membuat semua kemajuan ketika total dari lalu lintas sudah melebihi kapasitas),
+toleransi untuk lonjakan lalu lintas, dan penambahan latensi yang dihasilkan oleh antrian.
+
+### Permintaan yang dikecualikan (_Exempt Requests_)
+Beberapa permintaan dianggap cukup penting sehingga mereka tidak akan mengikuti
+salah satu batasan yang diberlakukan oleh fitur ini. Pengecualian ini untuk mencegah
+konfigurasi _flow control_ yang tidak terkonfigurasi dengan baik sehingga tidak benar-benar menonaktifkan server API.
+
+## Bawaan (_Defaults_)
```

Comment:
I think "bawaan" is generally understandable, but if you think not, feel free to keep this but with singular form
```suggestion
## Bawaan (_Default_)
```

Reply:
Just to keep consistency with the previous PR.